packages/charts/react-charting/src/components/HorizontalBarChartWithAxis/HorizontalBarChartWithAxis.base.tsx

```diff
@@ -1006,4 +1011,28 @@ export class HorizontalBarChartWithAxisBase
     const { chartTitle, data } = this.props;
     return (chartTitle ? `${chartTitle}. ` : '') + `Horizontal bar chart with ${data?.length || 0} bars. `;
   };
+
+  private _getOrderedYAxisLabels = () => {
+    const shouldOrderYAxisLabelsByCategoryOrder =
+      this._yAxisType === YAxisType.StringAxis && this.props.yAxisCategoryOrder !== 'default';
+    if (!shouldOrderYAxisLabelsByCategoryOrder) {
+      const reversedBars = [...this._points].reverse();
```

Comment:
could you add a comment here why we are reversing by default.

Reply:
done